text_output.go

```diff
@@ -77,10 +77,7 @@ func (o *textOutput) print(writer io.Writer, prefix string, skipFrames int, prin
 		errorOnLogging(err)
 	}
 	if printStack {
-		var b []byte
-		buf := bytes.NewBuffer(b)
-		getStack(buf, o.pc)
-		if _, err := buf.WriteTo(writer); err != nil {
+		if err := writeStack(writer, o.pc); err != nil {
```

Comment:
I know that this was actually old logic from golog.go, but it occurred to me that there's no need to allocate a new buffer here.